deploy/stop

```diff
@@ -3,6 +3,10 @@ set -euo pipefail
 
 . "$(dirname "${0}")/utils.sh"
 
+if has_namespace $APP_NAMESPACE_NAME; then
+  get_logs
```

Comment:
`get_error_logs`?

Reply:
If the test pass i will get follower logs as well.
And in addition to that i'm getting logs when i see problems i'm not sure that each log will contains error